skt/kerneltree.py

```diff
@@ -307,6 +329,15 @@ def merge_git_ref(self, uri, ref="master"):
         return (SKT_SUCCESS, head)
 
     def merge_patchwork_patch(self, uri):
+        """
+        Apply a patch from Patchwork (using git am)
+
+        Args:
+            uri: URL of patch on a Patchwork instance.
+
+        Raises:
+            PatchApplicationError in case the failed to apply patch
```

Comment:
Can we adjust the working a bit here and with `merge_patch_file` too? "... in case the patch failed to apply" or "... in case of patch application failure" would sound a lot better